File: Java/1060.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);
        
        double vet[] = new double[6];
        int positivo = 0;
        
        for (int i = 0; i < 6; i++) {
            vet[i] = leitor.nextDouble();
        }
        
        for (int i = 0; i < 6; i++) {
            if (vet[i] > 0) {
                positivo++;
            }
        }
        System.out.println(positivo + " valores positivos");
    }
 
}

File: Python/1060.py
# -*- coding: utf-8 -*-
lista = [ ]
positivo = 0

lista.append(float(input()))
lista.append(float(input()))
lista.append(float(input()))
lista.append(float(input()))
lista.append(float(input()))
lista.append(float(input()))

for n in lista:
    if n > 0:
        positivo += 1

print(positivo, "valores positivos")